src/Faq.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+namespace PrestaShop\ModuleLibFaq;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+
+/**
+ * Retrieve the FAQ of the module
+ */
+class Faq
+{
+    const BASE_URL = 'https://api.addons.prestashop.com/request/faq/';
+
+    /**
+     * @var Client
+     */
+    private $client;
+
+    /**
+     * @var Parameters
+     */
+    private $parameters;
+
+    /**
+     * Method to call in case of error
+     *
+     * @var callable|null
+     */
+    private $errorCallable;
+
+    /**
+     * @param string $moduleKey
+     * @param string $psVersion
+     * @param string $isoCode
+     */
+    public function __construct($moduleKey, $psVersion, $isoCode, array $options = [])
+    {
+        $this->parameters = (new Parameters())
+            ->setModuleKey($moduleKey)
+            ->setPsVersion($psVersion)
+            ->setIsoCode($isoCode);
+
+        // Allow client options to be customized
+        $options = array_merge_recursive([
+            'base_url' => self::BASE_URL,
```

Comment:
Avoid `cURL error 60: SSL certificate problem: unable to get local issuer certificate` on windows and others server without a cacert or missing php configuration.
```suggestion
            'base_url' => self::BASE_URL,
            'verify' => defined('_PS_CACHE_CA_CERT_FILE_') ? constant('_PS_CACHE_CA_CERT_FILE_') : true,
```